### ISSUE TITLE
Does Ungoogled-Chromium Do This? (Google Chrome Malware/Virus Scan)

### ISSUE THREAD

**Issue by ilikenwf** [Eloston/ungoogled-chromium#387]
Despite using firejail this may be another collection of domains to blacklist, or a blob to remove? Comments there on slashdot suggest the domains are "scorecard" related.

Very creepy that Google thinks it has the right to scan all the files on a given machine without consent.

https://yro.slashdot.org/story/18/04/02/2041254/chrome-is-scanning-files-on-your-computer-and-people-are-freaking-out

**Comment by xsmile** [comment]
From what I gathered from [1] Chrome Cleanup Tool consists of a Software Reporting component and a Cleanup Tool that is downloaded on-demand if a threat is found by Software Reporting.

The first one can be found in Google Chrome only but not in Chromium; check the page chromium:components to see if it is installed. Furthermore ungoogled-chromium prevents components from being updated. The second one heavily relies on the Safe Browsing service which is not available in ungoogled-chromium. Even if the tool was downloaded automatically somehow, although very unlikely since all Google URLs were removed to prevent any unwanted communication, it would not function properly.

1: https://cs.chromium.org/chromium/src/chrome_cleaner/README.md?q=UwS&sq=package:chromium&dr=C&l=5
